diff --git a/app/views/layouts/_meta.html.erb b/app/views/layouts/_meta.html.erb
index d54fe70e7..b8af58472 100644
--- a/app/views/layouts/_meta.html.erb
+++ b/app/views/layouts/_meta.html.erb
@@ -2,6 +2,7 @@
 
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1">
+<%#= favicon_link_tag "ohm_logo.ico" %>
 <% [57, 60, 72, 76, 114, 120, 144, 152, 180].each do |size| -%>
 <%= favicon_link_tag "apple-touch-icon-#{size}x#{size}.png", :rel => "apple-touch-icon", :sizes => "#{size}x#{size}", :type => "image/png" %>
 <% end -%>
@@ -12,6 +13,7 @@
 <%= favicon_link_tag "favicon-16x16.png", :rel => "icon", :sizes => "16x16", :type => "image/png" %>
 <%= tag.link :rel => "mask-icon", :href => asset_path("tab-icon.svg"), :color => "#7ebc6f" %>
 <%= tag.link :rel => "manifest", :href => asset_path("manifest.json") %>
+<%= tag.link :rel => "me", :href => Settings.mastodon_url %>
 <%= tag.meta :name => "msapplication-config", :content => asset_path("browserconfig.xml") %>
 <%= tag.meta :name => "msapplication-TileColor", :content => "#00a300" %>
 <%= tag.meta :name => "msapplication-TileImage", :content => image_path("mstile-144x144.png") %>
