diff --git a/app/views/layouts/map.html.erb b/app/views/layouts/map.html.erb
index 294f31761..ba3e2e76e 100644
--- a/app/views/layouts/map.html.erb
+++ b/app/views/layouts/map.html.erb
@@ -31,10 +31,11 @@
         <button type="button" class="btn-close position-absolute end-0 top-0 m-2 rounded-5 p-2" aria-label="<%= t("javascripts.close") %>"></button>
         <h2 class="me-4 text-break"><%= t "layouts.intro_header" %></h2>
         <p class="fs-6 fw-light"><%= t "layouts.intro_text" %></p>
-        <p class="fs-7 fw-light"><%= t "layouts.hosting_partners_2024_html",
-                                       :fastly => link_to(t("layouts.partners_fastly"), "https://www.fastly.com/"),
-                                       :corpmembers => link_to(t("layouts.partners_corpmembers"), "https://osmfoundation.org/wiki/Corporate_Members"),
-                                       :partners => link_to(t("layouts.partners_partners"), "https://hardware.openstreetmap.org/thanks/") %>
+        <p class="fs-6 fw-light"><%= t "layouts.hosting_partners_2024_html",
+                                       :devseed => link_to(t("layouts.partners_devseed"), "https://developmentseed.org/"),
+                                       :greeninfo => link_to(t("layouts.partners_greeninfo"), "https://greeninfo.org/"),
+                                       :osmf => link_to(t("layouts.partners_osmf"), "https://osmfoundation.org/"),
+                                       :osmus => link_to(t("layouts.partners_osmus"), "https://openstreetmap.us/") %>
         </p>
         <p class="fs-7 fw-light">
           <%= t "layouts.welcome_tou_notice_html",
