diff --git a/app/views/layouts/user_mailer.html.erb b/app/views/layouts/user_mailer.html.erb
index b597c3af0..8b5d4c9ed 100644
--- a/app/views/layouts/user_mailer.html.erb
+++ b/app/views/layouts/user_mailer.html.erb
@@ -12,7 +12,7 @@
             <tr>
               <td style="width: 30px; padding-top: 10px; padding-<%= style_right %>: 10px; padding-bottom: 10px; padding-<%= style_left %>: 0px">
                 <a href="<%= @root_url %>" target="_blank">
-                  <%= image_tag attachments["logo.png"].url, :alt => "OpenStreetMap", :title => "OpenStreetMap", :height => "30", :width => "30", :border => "0" %>
+                  <%= image_tag attachments["logo.png"].url, :alt => "OpenHistoricalMap", :title => "OpenHistoricalMap", :height => "30", :width => "30", :border => "0" %>
                 </a>
               </td>
               <%# the "width: 100%" here looks wrong, but I couldn't find a better way of making Outlook give this cell full width %>
