diff --git a/test/system/account_pd_declaration_test.rb b/test/system/account_pd_declaration_test.rb
index 5654a4efa..bfb2b88c7 100644
--- a/test/system/account_pd_declaration_test.rb
+++ b/test/system/account_pd_declaration_test.rb
@@ -8,36 +8,13 @@ class AccountPdDeclarationTest < ApplicationSystemTestCase
     sign_in_as(@user)
   end
 
-  test "can decline declaration if no declaration was made" do
-    visit account_pd_declaration_path
-
-    within_content_body do
-      assert_unchecked_field "I consider my contributions to be in the Public Domain"
-      assert_button "Confirm"
-
-      click_on "Confirm"
-
-      assert_no_text "You have also declared that you consider your edits to be in the Public Domain."
-    end
-  end
-
-  test "can confirm declaration if no declaration was made" do
-    visit account_pd_declaration_path
-
-    within_content_body do
-      assert_unchecked_field "I consider my contributions to be in the Public Domain"
-      assert_button "Confirm"
-
-      check "I consider my contributions to be in the Public Domain"
-      click_on "Confirm"
-
-      assert_text "You have also declared that you consider your edits to be in the Public Domain."
-    end
-  end
-
-  test "show disabled checkbox if declaration was made" do
-    @user.update(:consider_pd => true)
-
+  # OHM variant: contributions must be in the Public Domain so we've removed
+  #   test "can decline declaration if no declaration was made"
+  # and
+  #   test "can confirm declaration if no declaration was made"
+  #
+  # We've renamed this one & removed `@user.update(:consider_pd => true)`
+  test "show disabled checkbox & button since contributions must be in the Public Domain" do
     visit account_pd_declaration_path
 
     within_content_body do
