diff --git a/test/system/user_logout_test.rb b/test/system/user_logout_test.rb
index 3fa2723a7..eb8607c10 100644
--- a/test/system/user_logout_test.rb
+++ b/test/system/user_logout_test.rb
@@ -30,7 +30,7 @@ class UserLogoutTest < ApplicationSystemTestCase
     assert_no_content "Log In"
 
     visit logout_path
-    assert_content "Logout from OpenStreetMap"
+    assert_content "Logout from OpenHistoricalMap"
 
     click_on "Logout"
     assert_content "Log In"
@@ -41,7 +41,7 @@ class UserLogoutTest < ApplicationSystemTestCase
     assert_no_content "Log In"
 
     visit logout_path(:referer => "/traces")
-    assert_content "Logout from OpenStreetMap"
+    assert_content "Logout from OpenHistoricalMap"
 
     click_on "Logout"
     assert_content "Log In"
