diff --git a/.gitignore b/.gitignore
index d98d0b8a0..ff2cf0157 100644
--- a/.gitignore
+++ b/.gitignore
@@ -15,6 +15,8 @@ config/settings.local.yml
 config/settings/*.local.yml
 coverage
 database_consistency_*.txt
+doc
+i18n/data
 log
 node_modules
 public/assets
@@ -23,5 +25,12 @@ public/export
 storage
 tmp
 
+##local dev artifacts
+ohm-docker.env
+
 # docker-compose database directory
 docker-db-data
+
+/config/master.key
+i18n/data/
+backup.sql
