diff --git a/test/models/message_test.rb b/test/models/message_test.rb
index 53fc17fa5..dcb0562b9 100644
--- a/test/models/message_test.rb
+++ b/test/models/message_test.rb
@@ -157,13 +157,13 @@ class MessageTest < ActiveSupport::TestCase
     assert_equal "text", message.body_format
   end
 
-  def test_from_mail_prefix
+  def test_from_mail_prefix_ohm
     sender_user = create(:user)
     recipient_user = create(:user)
     mail = Mail.new do
       from "from@example.com"
       to "to@example.com"
-      subject "[OpenStreetMap] Test message"
+      subject "Test message"
       date Time.now.utc
       content_type "text/plain; charset=utf-8"
       body "This is a test & a message"
