diff --git a/vendor/assets/leaflet/leaflet.osm.js b/vendor/assets/leaflet/leaflet.osm.js
index 4bce7a623..ca867123c 100644
--- a/vendor/assets/leaflet/leaflet.osm.js
+++ b/vendor/assets/leaflet/leaflet.osm.js
@@ -29,6 +29,13 @@ L.OSM.CyclOSM = L.OSM.TileLayer.extend({
   }
 });
 
+L.OSM.Historical = L.OSM.TileLayer.extend({
+  options: {
+    url: 'http://www.openhistoricalmap.org/ohm_tiles/{z}/{x}/{y}.png',
+    maxZoom: 20
+  }
+});
+
 L.OSM.CycleMap = L.OSM.TileLayer.extend({
   options: {
     url: 'https://api.thunderforest.com/cycle/{z}/{x}/{y}{r}.png?apikey={apikey}',
