diff --git a/app/helpers/application_helper.rb b/app/helpers/application_helper.rb
index a591d1451..bc55850a3 100644
--- a/app/helpers/application_helper.rb
+++ b/app/helpers/application_helper.rb
@@ -45,7 +45,6 @@ module ApplicationHelper
       [export_path, t("layouts.header.export"), { :class => ["geolink"] }],
       [traces_path, t("layouts.header.gps_traces")],
       [diary_entries_path, t("layouts.header.user_diaries")],
-      [communities_path, t("layouts.header.communities")],
       [copyright_path, t("layouts.header.copyright")],
       [help_path, t("layouts.header.help")],
       [Settings.donation_url, t("layouts.header.donate"), { :target => :new }],
