diff --git a/app/views/site/fixthemap.html.erb b/app/views/site/fixthemap.html.erb
index dcde4f207..53c974406 100644
--- a/app/views/site/fixthemap.html.erb
+++ b/app/views/site/fixthemap.html.erb
@@ -23,8 +23,9 @@
 
 <h2><%= t ".other_concerns.title" %></h2>
 <p>
-  <%= t ".other_concerns.concerns_html", :copyright_link => link_to(t(".other_concerns.copyright"), copyright_path),
-                                         :working_group_link => link_to(t(".other_concerns.working_group"), t(".other_concerns.working_group_url")) %>
+  <%= t ".other_concerns.concerns_ohm_html",
+        :copyright_link => link_to(t(".other_concerns.copyright"), copyright_path),
+        :contact_link => link_to(t(".other_concerns.contact"), t(".other_concerns.contact_url")) %>
 </p>
 
 <%= render "any_questions" %>
