diff --git a/app/views/site/id.html.erb b/app/views/site/id.html.erb
index f21b5abfa..26ecc23ea 100644
--- a/app/views/site/id.html.erb
+++ b/app/views/site/id.html.erb
@@ -14,7 +14,7 @@
    end
    data[:locale] = ID::LOCALES.preferred(preferred_languages).to_s
    data[:theme] = current_user&.preferred_color_scheme(:editor, :site)
-   data[:asset_map] = assets("@openstreetmap/id").to_json %>
+   data[:asset_map] = assets("@openhistoricalmap/id").to_json %>
 <%= tag.div "", :id => "id-container", :data => data %>
 </body>
 </html>
