diff --git a/app/views/site/welcome.html.erb b/app/views/site/welcome.html.erb
index 6dabde763..ae860f750 100644
--- a/app/views/site/welcome.html.erb
+++ b/app/views/site/welcome.html.erb
@@ -13,7 +13,7 @@
 <div class="row">
   <div class="col-sm">
     <%= inline_svg_tag "welcome/check.svg", :class => "d-block mx-auto" %>
-    <p><%= t ".whats_on_the_map.on_the_map_html", :real_and_current => tag.em(t(".whats_on_the_map.real_and_current")) %></p>
+    <p><%= t ".whats_on_the_map.on_the_map_ohm_html", :past_and_present => tag.em(t(".whats_on_the_map.past_and_present")) %></p>
   </div>
   <div class="col-sm">
     <%= inline_svg_tag "welcome/cross.svg", :class => "d-block mx-auto" %>
