diff --git a/config/routes.rb b/config/routes.rb
index e63bed58e..69b67d566 100644
--- a/config/routes.rb
+++ b/config/routes.rb
@@ -195,7 +195,6 @@ OpenStreetMap::Application.routes.draw do
   get "/help" => "site#help"
   get "/about/:about_locale" => "site#about"
   get "/about" => "site#about"
-  get "/communities" => "site#communities"
   get "/history" => "changesets#index"
   get "/history/feed" => "changesets#feed", :defaults => { :format => :atom }
   scope "/history" do
@@ -232,6 +231,9 @@ OpenStreetMap::Application.routes.draw do
   get "/create-account.html", :to => redirect(:path => "/user/new")
   get "/forgot-password.html", :to => redirect(:path => "/user/forgot-password")
 
+  # ohm
+  get "/privacy-policy" => "site#ohm_privacy_policy"
+
   # omniauth
   get "/auth/failure" => "users#auth_failure"
   match "/auth/:provider/callback" => "users#auth_success", :via => [:get, :post], :as => :auth_success
