diff --git a/test/controllers/diary_entries_controller_test.rb b/test/controllers/diary_entries_controller_test.rb
index efdc59a68..43a334578 100644
--- a/test/controllers/diary_entries_controller_test.rb
+++ b/test/controllers/diary_entries_controller_test.rb
@@ -10,7 +10,7 @@ class DiaryEntriesControllerTest < ActionDispatch::IntegrationTest
     # Create the default language for diary entries
     create(:language, :code => "en")
     # Stub nominatim response for diary entry locations
-    stub_request(:get, %r{^https://nominatim\.openstreetmap\.org/reverse\?})
+    stub_request(:get, %r{^https://nominatim\.openhistoricalmap\.org/reverse\?})
       .to_return(:status => 404)
   end
 
@@ -664,7 +664,7 @@ class DiaryEntriesControllerTest < ActionDispatch::IntegrationTest
       assert_dom "> @content", ActionController::Base.helpers.image_url("osm_logo_256.png", :host => root_url)
     end
     assert_dom "head meta[property='og:image:alt']" do
-      assert_dom "> @content", "OpenStreetMap logo"
+      assert_dom "> @content", "OpenHistoricalMap logo"
     end
   end
 
@@ -734,7 +734,7 @@ class DiaryEntriesControllerTest < ActionDispatch::IntegrationTest
       assert_dom "> @content", ActionController::Base.helpers.image_url("osm_logo_256.png", :host => root_url)
     end
     assert_dom "head meta[property='og:image:alt']" do
-      assert_dom "> @content", "OpenStreetMap logo"
+      assert_dom "> @content", "OpenHistoricalMap logo"
     end
   end
 
