diff --git a/db/structure.sql b/db/structure.sql
index da1d403ad..d8ade4c19 100644
--- a/db/structure.sql
+++ b/db/structure.sql
@@ -1699,7 +1699,7 @@ CREATE TABLE public.users (
     languages character varying,
     status public.user_status_enum DEFAULT 'pending'::public.user_status_enum NOT NULL,
     terms_agreed timestamp without time zone,
-    consider_pd boolean DEFAULT false NOT NULL,
+    consider_pd boolean DEFAULT true NOT NULL,
     auth_uid character varying,
     preferred_editor character varying,
     terms_seen boolean DEFAULT false NOT NULL,
@@ -3811,6 +3811,7 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20240618193051'),
 ('20240605134916'),
 ('20240405083825'),
+('20240401010024'),
 ('20240307181018'),
 ('20240307180830'),
 ('20240228205723'),
