diff --git a/db/migrate/20240405083825_add_creator_index_to_user_blocks.rb b/db/migrate/20240405083825_add_creator_index_to_user_blocks.rb
index 9d08caccb..7a6a82674 100644
--- a/db/migrate/20240405083825_add_creator_index_to_user_blocks.rb
+++ b/db/migrate/20240405083825_add_creator_index_to_user_blocks.rb
@@ -4,6 +4,8 @@ class AddCreatorIndexToUserBlocks < ActiveRecord::Migration[7.1]
   disable_ddl_transaction!
 
   def change
-    add_index :user_blocks, [:creator_id, :id], :algorithm => :concurrently
+    unless index_exists?(:user_blocks, [:creator_id, :id])
+      add_index :user_blocks, [:creator_id, :id], algorithm: :concurrently
+    end
   end
-end
+end
\ No newline at end of file
