diff --git a/app/models/social_link.rb b/app/models/social_link.rb
index 76d61644b..c0b41a709 100644
--- a/app/models/social_link.rb
+++ b/app/models/social_link.rb
@@ -43,6 +43,9 @@ class SocialLink < ApplicationRecord
     :medium => %r{\Ahttps?://(?:www\.)?medium\.com/@([a-zA-Z0-9_]+)},
     :ogf => %r{\Ahttps?://wiki\.opengeofiction\.net/index\.php/User:([a-zA-Z0-9_-]+)},
     :ohm => %r{\Ahttps?://(?:www\.)?openhistoricalmap\.org/user/(\S+)},
+    :ohm_forum => %r{\Ahttps?://forum\.openhistoricalmap\.org/u/(\S+)},
+    :ohm_tasking_manager => %r{\Ahttps?://tasks\.openhistoricalmap\.org/users/(\S+)},
+    :osm => %r{\Ahttps?://(?:www\.)?openstreetmap\.org/user/(\S+)},
     :osm_forum => %r{\Ahttps?://community\.openstreetmap\.org/u/(\S+)},
     :osm_wiki => %r{\Ahttps?://wiki\.openstreetmap\.org/wiki/User:([a-zA-Z0-9_-]+)},
     :quora => %r{\Ahttps?://(?:www\.)?quora\.com/profile/([a-zA-Z0-9_-]+)},
