diff --git a/app/assets/javascripts/index_modules/history.js b/app/assets/javascripts/index_modules/history.js
index 1f1bb62ec..7722b909d 100644
--- a/app/assets/javascripts/index_modules/history.js
+++ b/app/assets/javascripts/index_modules/history.js
@@ -297,6 +297,8 @@ export default function (map) {
     map.on("moveend", moveEndListener);
     map.on("zoomend", zoomEndListener);
     loadFirstChangesets();
+
+    addOpenHistoricalMapTimeSlider(map);
   };
 
   page.unload = function () {
