diff --git a/app/assets/javascripts/index_modules/history.js b/app/assets/javascripts/index_modules/history.js
index 8b64a4432..dd43f6219 100644
--- a/app/assets/javascripts/index_modules/history.js
+++ b/app/assets/javascripts/index_modules/history.js
@@ -48,9 +48,9 @@ OSM.History = function (map) {
 
     changesetIntersectionObserver = new IntersectionObserver((entries) => {
       let closestTargetToTop,
-          closestDistanceToTop = Infinity,
-          closestTargetToBottom,
-          closestDistanceToBottom = Infinity;
+        closestDistanceToTop = Infinity,
+        closestTargetToBottom,
+        closestDistanceToBottom = Infinity;
 
       for (const entry of entries) {
         const id = $(entry.target).data("changeset")?.id;
@@ -175,7 +175,7 @@ OSM.History = function (map) {
     if (isHistory) {
       setBboxFetchData(data);
       const feedLink = $("link[type=\"application/atom+xml\"]"),
-            feedHref = feedLink.attr("href").split("?")[0];
+        feedHref = feedLink.attr("href").split("?")[0];
       feedLink.attr("href", feedHref + "?" + data);
     }
 
@@ -295,6 +295,8 @@ OSM.History = function (map) {
     map.on("moveend", moveEndListener);
     map.on("zoomend", zoomEndListener);
     loadFirstChangesets();
+
+    addOpenHistoricalMapTimeSlider(map);
   };
 
   page.unload = function () {
