diff --git a/app/assets/javascripts/index_modules/note.js b/app/assets/javascripts/index_modules/note.js
index 006d7be31..f8f37058e 100644
--- a/app/assets/javascripts/index_modules/note.js
+++ b/app/assets/javascripts/index_modules/note.js
@@ -13,6 +13,7 @@ export default function (map) {
 
   page.init = function (path, id) {
     initialize(path, id);
+    addOpenHistoricalMapTimeSlider(map);
   };
 
   function initialize(path, id, skipMoveToNote) {
