diff --git a/app/assets/javascripts/index_modules/note.js b/app/assets/javascripts/index_modules/note.js
index ad3844559..b2fdbb085 100644
--- a/app/assets/javascripts/index_modules/note.js
+++ b/app/assets/javascripts/index_modules/note.js
@@ -1,6 +1,6 @@
 OSM.Note = function (map) {
   const content = $("#sidebar_content"),
-        page = {};
+    page = {};
 
   page.pushstate = page.popstate = function (path, id) {
     OSM.loadSidebarContent(path, function () {
@@ -13,14 +13,15 @@ OSM.Note = function (map) {
 
   page.load = function (path, id) {
     initialize(path, id);
+    addOpenHistoricalMapTimeSlider(map);
   };
 
   function initialize(path, id, skipMoveToNote) {
     content.find("button[name]").on("click", function (e) {
       e.preventDefault();
       const { url, method } = $(e.target).data(),
-            name = $(e.target).attr("name"),
-            data = new URLSearchParams();
+        name = $(e.target).attr("name"),
+        data = new URLSearchParams();
       content.find("button[name]").prop("disabled", true);
 
       if (name !== "subscribe" && name !== "unsubscribe") {
