diff --git a/app/views/changesets/index.atom.builder b/app/views/changesets/index.atom.builder
index 237a4ab63..9c2b58d70 100644
--- a/app/views/changesets/index.atom.builder
+++ b/app/views/changesets/index.atom.builder
@@ -12,7 +12,7 @@ atom_feed(:language => I18n.locale, :schema_date => 2009,
   feed.logo image_url("osm_logo_100.png")
 
   feed.rights :type => "xhtml" do |xhtml|
-    xhtml.a "Open Data Commons Open Database License", :href => "https://opendatacommons.org/licenses/odbl/"
+    xhtml.a t(".feed.cc0"), :href => t(".feed.cc0_url")
   end
 
   @changesets.items.each do |changeset|
