diff --git a/test/integration/login_test.rb b/test/integration/login_test.rb
index e3dc0368d..9bfae22b9 100644
--- a/test/integration/login_test.rb
+++ b/test/integration/login_test.rb
@@ -116,7 +116,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -128,7 +128,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -140,7 +140,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -273,7 +273,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -285,7 +285,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -297,7 +297,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -412,7 +412,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -586,7 +586,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -729,7 +729,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -868,7 +868,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -1007,7 +1007,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
@@ -1146,7 +1146,7 @@ class LoginTest < ActionDispatch::IntegrationTest
     assert_template "sessions/new"
     assert_select "span.username", false
     assert_select "div.alert.alert-danger", /your account has been suspended/ do
-      assert_select "a[href='mailto:openstreetmap@example.com']", "support"
+      assert_select "a[href='mailto:dev@openhistoricalmap.org']", "support"
     end
   end
 
