diff --git a/app/controllers/users_controller.rb b/app/controllers/users_controller.rb
index 5bb013427..c1cf8a380 100644
--- a/app/controllers/users_controller.rb
+++ b/app/controllers/users_controller.rb
@@ -131,11 +131,11 @@ class UsersController < ApplicationController
 
     provider = auth_info[:provider]
     uid = auth_info[:uid]
-    name = auth_info[:info][:name]
+    name = provider.to_s == "openstreetmap" ? auth_info.dig(:extra, :raw_info, "preferred_username") : auth_info[:info][:name]
     email = auth_info[:info][:email]
 
     email_verified = case provider
-                     when "google", "apple", "facebook", "microsoft", "github", "wikipedia"
+                     when "google", "apple", "facebook", "microsoft", "github", "wikipedia", "openstreetmap"
                        true
                      else
                        false
