diff --git a/app/assets/javascripts/edit/id.js.erb b/app/assets/javascripts/edit/id.js.erb
index 07cc136ce..7750c6db9 100644
--- a/app/assets/javascripts/edit/id.js.erb
+++ b/app/assets/javascripts/edit/id.js.erb
@@ -1,6 +1,6 @@
 $(function () {
   const id = $("#id-embed"),
-        idData = id.data();
+    idData = id.data();
 
   if (!idData.configured) {
     OSM.showAlert(OSM.i18n.t("javascripts.edit.id_not_configured.title"),
