diff --git a/app/assets/javascripts/leaflet.share.js b/app/assets/javascripts/leaflet.share.js
index 5896eed9e..2561f7348 100644
--- a/app/assets/javascripts/leaflet.share.js
+++ b/app/assets/javascripts/leaflet.share.js
@@ -43,7 +43,7 @@ L.OSM.share = function (options) {
 
     marker.on("dragend", movedMarker);
     map.on("move", movedMap);
-    map.on("moveend baselayerchange overlayadd overlayremove", update);
+    map.on("moveend layeradd layerremove baselayerchange overlayadd overlayremove", update);
 
     $ui
       .on("show", shown)
@@ -155,10 +155,10 @@ L.OSM.share = function (options) {
 
       $("#embed_html").val(
         "<iframe width=\"425\" height=\"350\" src=\"" +
-          escapeHTML(OSM.SERVER_PROTOCOL + "://" + OSM.SERVER_URL + "/export/embed.html?" + params) +
-          "\" style=\"border: 1px solid black\"></iframe><br/>" +
-          "<small><a href=\"" + escapeHTML(map.getUrl(marker)) + "\">" +
-          escapeHTML(OSM.i18n.t("javascripts.share.view_larger_map")) + "</a></small>");
+        OSM.EMBED_SERVER_URL + location.hash + "&bbox=" + params.bbox +
+        "\" style=\"border: 1px solid black\"></iframe><br/>" +
+        "<small><a href=\"" + escapeHTML(map.getUrl(marker)) + "\">" +
+        escapeHTML(OSM.i18n.t("javascripts.share.view_larger_map")) + "</a></small>");
 
       // Geo URI
 
@@ -192,10 +192,14 @@ L.OSM.share = function (options) {
       $("#mapnik_image_width").text(mapWidth);
       $("#mapnik_image_height").text(mapHeight);
 
-      const canDownloadImage = Boolean(layer && layer.options.canDownloadImage);
+      const layerKeys = new Map([
+        ["mapnik", "standard"],
+        ["cyclemap", "cycle_map"],
+        ["transportmap", "transport_map"]
+      ]);
 
-      $("#mapnik_image_layer").text(canDownloadImage ? layer.options.name : "");
-      $("#map_format").val(canDownloadImage ? layer.options.layerId : "");
+      $("#mapnik_image_layer").text(layerKeys.has(layer) ? OSM.i18n.t(`javascripts.map.base.${layerKeys.get(layer)}`) : "");
+      $("#map_format").val(layer);
 
       $("#map_zoom").val(map.getZoom());
       $("#mapnik_lon").val(map.getCenter().lng);
@@ -203,9 +207,18 @@ L.OSM.share = function (options) {
       $("#map_width").val(mapWidth);
       $("#map_height").val(mapHeight);
 
-      $("#export-image").toggle(canDownloadImage);
-      $("#export-warning").toggle(!canDownloadImage);
-      $("#mapnik_scale_row").toggle(canDownloadImage && layer.options.layerId === "mapnik");
+      if (["cyclemap", "transportmap"].includes(map.getMapBaseLayerId())) {
+        $("#export-image").show();
+        $("#mapnik_scale_row").hide();
+        $("#export-warning").hide();
+      } else if (map.getMapBaseLayerId() === "mapnik") {
+        $("#export-image").show();
+        $("#mapnik_scale_row").show();
+        $("#export-warning").hide();
+      } else {
+        $("#export-image").hide();
+        $("#export-warning").show();
+      }
     }
 
     function select() {
