diff --git a/app/assets/javascripts/router.js b/app/assets/javascripts/router.js
index f7ddb571c..36c38112a 100644
--- a/app/assets/javascripts/router.js
+++ b/app/assets/javascripts/router.js
@@ -198,7 +198,7 @@ OSM.Router = function (map, rts) {
     document.body.removeChild(link);
   };
 
-  map.on("moveend baselayerchange overlayadd overlayremove", router.updateHash);
+  map.on("moveend layeradd baselayerchange overlayadd overlayremove", router.updateHash);
   $(window).on("hashchange", router.hashUpdated);
 
   return router;
