diff --git a/test/lib/rich_text_test.rb b/test/lib/rich_text_test.rb
index 2505ddb8c..5722e5c8d 100644
--- a/test/lib/rich_text_test.rb
+++ b/test/lib/rich_text_test.rb
@@ -246,7 +246,7 @@ class RichTextTest < ActiveSupport::TestCase
       r = RichText.new("text", "foo repl.example.com/some/path?query=te<st&limit=20>10#result12 bar")
       assert_html r do
         assert_dom "a", :count => 1, :text => "repl.example.com/some/path?query=te<st&limit=20>10#result12" do
-          assert_dom "> @href", "http://replace-me.example.com/some/path?query=te<st&limit=20>10#result12"
+          assert_dom "> @href", "https://replace-me.example.com/some/path?query=te<st&limit=20>10#result12"
           assert_dom "> @rel", "nofollow noopener noreferrer"
         end
       end
@@ -330,7 +330,7 @@ class RichTextTest < ActiveSupport::TestCase
       r = RichText.new("text", "foo wiki.example.com/Tag:surface%3Dmetal bar")
       assert_html r do
         assert_dom "a", :count => 1, :text => "wiki.example.com/Tag:surface%3Dmetal" do
-          assert_dom "> @href", "http://replace-me-wiki.example.com/Tag:surface%3Dmetal"
+          assert_dom "> @href", "https://replace-me-wiki.example.com/Tag:surface%3Dmetal"
           assert_dom "> @rel", "nofollow noopener noreferrer"
         end
       end
@@ -355,8 +355,8 @@ class RichTextTest < ActiveSupport::TestCase
     with_settings(:linkify => { :detection_rules => [{ :patterns => ["@(?<username>\\w+)"], :path_template => "user/\\k<username>" }] }) do
       r = RichText.new("text", "foo @example bar")
       assert_html r do
-        assert_dom "a", :count => 1, :text => "http://test.host/user/example" do
-          assert_dom "> @href", "http://test.host/user/example"
+        assert_dom "a", :count => 1, :text => "https://test.host/user/example" do
+          assert_dom "> @href", "https://test.host/user/example"
           assert_dom "> @rel", "nofollow noopener noreferrer"
         end
       end
